### PR TITLE
DE299473 unable to restart qubeship after machine reboots

### PR DESCRIPTION
**Pull request by erioness1125**

### CODE REVIEW

**Comment by erioness1125** [comment]
@nidve01 @prabhuinbarajan @qubeomar @qubeosman 
To reviewers:

Problem statement:
Qubeship couldn't be fully restarted after the installation was completed.

Reason(s):
1. `qube-consul` service couldn't recover its last state (leader node election failure), causing the tenant and auth APIs failure.
1. `qube_builder` service couldn't start due to the `qube-consul` service failure.

Fix:
- run `qube-consul` in development mode: this is a workaround, making Consul always initialize itself. This does NOT persist any data, meaning the tenant data is going to be lost every time Qubeship restarts. Hence the qubebuilder should always log in to the system as the last step of `run.sh` to populate the tenant data.
> Note: We plan to re-write the tenant API to use different database, then Qubeship will have nearly zero-dependency to Consul.

**Comment by erioness1125** [comment]
Recommended way to test:

1. switch to community_beta branch and complete installation.
1. kill the containers (ex: `docker stop $(docker ps -aq)` or `./down.sh`)
1. switch to this branch and run `run.sh`

File: README.md [hunk 1]
```diff
@@ -86,6 +86,16 @@ Now, it is time to try qubeship. Please follow the <a href="https://github.com/Q
 
 ----
 
+## Restart
+> Note: make sure NOT TO HAVE `CONF_SERVER_ADDR` and/or `CONF_SERVER_TOKEN` in `.client_env` file!
+
+**Run** the following from **bootstrap** folder
+<pre> 
+$ ./run.sh --username <i>your_github_username</i> --password --github-host your_github_enterprise_url
```
Comment (nidve01) [inline README.md]: Could you add the below text to emulate the terminal prompt with bootstrap as current directory. Please refer to uninstall or other commands

```
mac1234:bootstrap user$

````

**Comment by nidve01** [comment]
verified the upgrade path (with your Recommended way to test),  working expected. Did any one verified the fresh install go through successfully?

**Pull request merged**